source/ways-of-working/deploying-an-app/java/java.html.md.erb

```diff
@@ -0,0 +1,115 @@
+---
+title: Deploy an Sample Java Application
+last_reviewed_on: 2021-11-19
+review_in: 6 months
+weight: 6
+---
+
+# <%= current_page.data.title %>
+
+## Overview
+This guide walk-through the process of deploying an Sample Java application in Cloud Platform.
+
+Before working through the tutorial, we recommend reading following pages and assume you have all the required accounts.
+
+- https://hmcts.github.io/onboarding
+- https://hmcts.github.io/ways-of-working/
+
+This guide uses a pre-configured application from an template which can be deployed from [Backstage](https://backstage.platform.hmcts.net/). Application is deployed to AKS(kubernetes) cluster inside `labs` namespace in sbox/sandbox environment.
+
+## Prerequisites
+
+This guide assumes the following are installed on your local system.
+
+- Docker
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [Helm](https://helm.sh/docs/intro/install/)
+
+## Steps
+
+1) Access [Backstage](https://backstage.platform.hmcts.net/create) to create a new github repo containing application template. Please make sure name of  Github repo  starts with `labs*`. This enables Jenkins build tool to organize the folder based on Github repo names.
+
+2) Create `Jenkinsfile_parameterized`  file in the repo with following contents.
+
+	#!groovy
+
+	@Library("Infrastructure")
+
+	def type = "java"
+	def product = "labs"
+	def component = ""   #githubreponame without "labs" prefix
+
+	withPipeline(type, product, component) {
+	}
+
+
+3) Create Helm values file with following contents.
+
+   CFT: values.sandbox.template.yaml
+
+   SDS: values.sbox.template.yaml
+
+
+	java:
+	  # Don't modify below here
+	  image: ${IMAGE_NAME}
+	  ingressHost: ${SERVICE_FQDN}
+
+
+4) Get the PR reviewed and merged.
```

Comment:
no need to get the PR reviewed just merge

avoid acronyms as well if they aren't needed

Reply:
I don't think developers has permission to merge the changes by default